app/frontend/src/components/Basics/_spacing.scss

```diff
@@ -23,20 +23,25 @@ $padding-abbrs: (
   "px": padding-right padding-left,
   "py": padding-top padding-bottom,
 );
-$spacing-abbrs: map.merge($margin-abbrs, $padding-abbrs);
-$spacing-sizes: 0, 1, 2, 3, 4, 5, 6;
+$gap-abbrs: (
+  "gap": gap,
```

Comment:
I'm thinking these will be easier to use if they're named "g, gx, gy". Thoughts?

Reply:
I always wondered why margins and paddings have short utility class names like `p-3`, `pt-5`, `mx-2`, `mb-4`, while gap utilities use the word 'gap' (maybe because it's already short to begin with?). Personally, I have no preference but CSS frameworks like Bootstrap and Tailwind both use gap class names like `gap-1`, `gap-x-2` and `gap-y-3` so I thought maybe that's the convention/industry standard 🤔

Reply:
Hmm, I know the 12 column system is a de-facto standard, but I've never heard of the class names being a standard. Curious to know more :hmm:

Reply:
Maybe there's no standard? I have no idea. I was referencing [bootstrap](https://getbootstrap.com/docs/5.0/utilities/spacing/#gap) and [tailwindCSS](https://tailwindcss.com/docs/gap).